src/index.js

```diff
@@ -438,8 +434,8 @@ export var app = function(props, enhance) {
 
   var setState = function(newState) {
     if (state !== newState) {
-      if (subscriptions) nextTask(subscribe)
-      if (view && !lock) nextFrame(render, (lock = true))
+      if (subscriptions) subscribe((state = newState))
```

Comment:
since subscriptions can immediately dispatch actions (dispatch internally has to use latest state), 
so state has to be assigned before subscriptions initialized

Reply:
Why not
```suggestion
      state = newState
      if (subscriptions) subscribe(state)
```
Or we should not update the state if there are no subscriptions and it is fine?

Reply:
Yup, that's how I refactored it after merging here. 👍